meilisync-healthcheck.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+tables=$(meilisync check 2>&1 | grep 'ERROR' | awk -F '"' '{print $2}' | awk -F '.' '{print $2}')
+args=""
+if echo "$tables" | grep -q "\w"; then
+    for table in $tables; do
+        args="$args -t $table"
+    done
+    meilisync refresh $args 2> /dev/null
```

Comment:
I tested locally with a database that had an index for a table with 100k rows, and when setting the refresh size to be 100, meaning a thousand inserts would have to take place, after 10 minutes it was still not done. I then tried with size of 1000 and it finished in 10 seconds. I know it is not exactly scalable, since a size of 1000 running 100 inserts would still be slower than a size of 1 with 100 insert, but this would indicate that having 100 times the size be the amount of rows for an index is still within a reasonable time. But even with this it might eat up a lot of RAM and processing power, so I will just remove the healthcheck and just use it for myself when running locally.